Source/SpatialGDK/Private/SpatialActorChannel.cpp

```diff
@@ -541,6 +544,38 @@ void USpatialActorChannel::PreReceiveSpatialUpdate(UObject* TargetObject)
 
 void USpatialActorChannel::PostReceiveSpatialUpdate(UObject* TargetObject, const TArray<UProperty*>& RepNotifies)
 {
+	const USpatialTypeBinding* Binding = SpatialNetDriver->GetSpatialInterop()->GetTypeBindingByClass(Actor->GetClass());
+	if (Binding)
+	{
+		worker::Map<worker::ComponentId, worker::InterestOverride> OldInterestOverrideMap = InterestOverrideMap;
+		InterestOverrideMap = Binding->GetInterestOverrideMap(SpatialNetDriver->GetNetMode() == NM_Client, Actor->GetNetConnection() != nullptr);
```

Comment:
`SpatialNetDriver->GetNetMode() == NM_Client`

How about in the eventual non-authoritative server scenario? i.e. overlap.

Reply:
How this currently works is like this: servers are always interested in everything, clients are never interested in handover data, and additionaly clients with nonowned actors are not interested in SingleClientRepData. I think this functionality is okay even in the case of non-authoritative servers, as they will need the data.